server/src/main/java/com/sauron/model/entities/Bank.java

```diff
@@ -0,0 +1,38 @@
+package com.sauron.model.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "BANKS")
+public class Bank {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, updatable = false, nullable = false)
+    private Long id;
+    private String name;
```

Comment:
I guess it should be at least unique, non-nullable and non-updatable.

Reply:
Unique, non-nullable: sure. 
Non-updatable: not so sure.